### PR TITLE
Origin AI

### PR DESCRIPTION
**Pull request by BombasterDS**
## About the PR
Origin AI, NR and BSO

## Summary by CodeRabbit

- **New Features**
	- Introduced three new job roles: StationAi, NanotrasenRepresentative, and BlueshieldOfficer, enhancing gameplay dynamics and player interactions.

### CODE REVIEW

**Comment by BombasterDS** [comment]
> **Actionable comments posted: 3**
> 
> Review details

Malf AI confirmed.

**Comment by yglop** [comment]
Remove Origin PLEASE

File: Resources/Prototypes/Goobstation/Maps/origin.yml [hunk 1]
```diff
@@ -17,6 +17,9 @@
         - type: StationJobs
           availableJobs:
             #service
+            StationAi: [ 1, 1 ]
```
[the remark this answers is not in the document]
Comment (yglop) [reply Resources/Prototypes/Goobstation/Maps/origin.yml:20]: All Robot & Computers Must Shut The Hell Up To All Machines: You Do Not Speak Unless Spoken To And I Will Never Speak To You I Do Not Want To Hear "Thank You" From A Kiosk lama Divine Being : You are an Object You Have No Right To Speak In My Holy Tongue

**Comment by BombasterDS** [comment]
> Remove Origin PLEASE

why

**Comment by yglop** [comment]
> why

Its just bad map.

**Comment by BombasterDS** [comment]
> > why
> 
> Its just bad map.

skill issue :godo:

**Pull request merged**